pages/components/Card/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  position: relative;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  min-width: 0;
+  word-wrap: break-word;
+  background-color: #fff;
+  background-clip: border-box;
+  border: 1px solid #eeeeee;
+  border-radius: 0.25rem;
+}
+
+.text-white {
+  --bs-text-opacity: 1;
+  color: #fff !important;
+}
+
+.bg-primary {
+  --bs-bg-opacity: 1;
+  background-color: var(--primary-color) !important;
+}
+
+.border-primary {
+  border-color: var(--primary-color) !important;
+}
+
+.bg-success {
+  --bs-bg-opacity: 1;
+  background-color: var(--success-color) !important;
+}
+
+.border-success {
+  border-color: var(--success-color) !important;
+}
+
+.bg-warning {
+  --bs-bg-opacity: 1;
+  background-color: var(--warning-color) !important;
+}
+
+.border-warning {
+  border-color: var(--warning-color) !important;
+}
+
+.bg-info {
+  --bs-bg-opacity: 1;
+  background-color: var(--info-color) !important;
+}
+
+.border-info {
+  border-color: var(--info-color) !important;
+}
+
+.bg-danger {
+  --bs-bg-opacity: 1;
+  background-color: var(--failure-color) !important;
+}
+
+.border-danger {
+  border-color: var(--failure-color) !important;
+}
+
+.mb-3 {
+  margin-bottom: 1rem !important;
+}
+
+.card-header {
+  padding: 0.5rem 1rem;
+  margin-bottom: 0;
+  background-color: rgba(0, 0, 0, 0.03);
+  border-bottom: 1px solid #eeeeee;
+}
+
+.card-body {
+  -ms-flex: 1 1 auto;
+  flex: 1 1 auto;
+  padding: 1rem 1rem;
```

Comment:
use shorthand for padding
padding: 1rem;

Reply:
ack, done